LnkParse3/extra_data.py

```diff
@@ -36,10 +37,15 @@ def _iter(self):
             if cls:
                 yield cls(indata=data, cp=self.cp)
 
+        # If there is data following the Terminal Block, we should take note of it and tell the user.
+        if len(rest) > 4 and rest[:4] < b"\x00\x00\x00\x04":
```

Comment:
> In terminal.py, my documentation says BlockSignature == 0x00000000 - 0x00000003, and I was wondering, based on endianness, if it was wrong and should be BlockSignature == 0x00000000 - 0x03000000? Same question would apply, but with more side effects in ExtraData._iter() with if len(rest) > 4 and rest[:4] < b"\x00\x00\x00\x04":, if that check is wrong based on the endianness. I modified it so that it would cover more than only 0x00, but it may cover WAY more now.

Your documentation is correct. About attached line, to be sure, we can do the same what we do with signature - parse it as an integer.
```python
unpack("<I", rest[:4])[0] < 0x00000004
```